### PR TITLE
PRMT-4706

### PR DESCRIPTION
**Pull request by MohammadIqbalAD-NHS**

### CODE REVIEW

File: gocd/deploy.pipeline.gocd.yml [hunk 1]
```diff
@@ -61,6 +51,48 @@ common:
...
+            tasks:
+              - exec:
+                  command: /bin/bash
+                  arguments:
+                    - -c
+                    - ./tasks tf_plan create
+    - deploy:
+        approval:
+          type: manual
+          allow_only_on_success: true
```
Comment (MohammadIqbalAD-NHS) [inline gocd/deploy.pipeline.gocd.yml:80]: This is the manual approval step.

Comment (MohammadIqbalAD-NHS) [inline gocd/deploy.pipeline.gocd.yml:80]: This is the manual approval step.

**Comment by martin-nhs** [comment]
Is this still required @MohammadIqbalAD-NHS - or can we close this PR?

**Comment by chrisbloe-nhse** [comment]
Closing as this has been superseded.

**Pull request closed without merging**